public/app/shared/filters.js

```diff
@@ -75,18 +75,19 @@
         return $sce.trustAsHtml(value);
       };
     }])
-    .filter('isPublished', ['$rootScope', 'Authorizer',
-      function($rootScope, Authorizer) {
-        return function(value) {
-          var out = [];
-          for (var i in value) {
-            if (value[i].status === 'published' ||
-              Authorizer.canAccess('administrate-motion'))
-              out.push(value[i]);
-          }
-          return out;
-        };
-      }])
+    //  deprecated
```

Comment:
Better to just remove this code, git tracks old stuff if we wanted to look back at it